### ISSUE TITLE
Missing colors in various places

### ISSUE THREAD

**Issue by SilverMarcs** [MrSpidercat/Matu#4]
**Describe the issue**
Color isn't applied to the places mentioned below. Probably other places too i missed.

**Where is this issue present**
Please list all places this missing/unchanged color is present:
1. Chat background
2. Server channel list
3. Server list
4. Text field
5. Bottom navigation panel
6. Appbar

**Expected behavior**
Discord og colors should be replaced by system theme ones on a12.

**Screenshots**
![Screenshot_20220216-065945_Aliucord](https://user-images.githubusercontent.com/77480421/154176028-dd39220f-eff9-4b54-99a9-746c0be18a80.png)
![Screenshot_20220216-065928_Aliucord](https://user-images.githubusercontent.com/77480421/154176029-dfe4ed37-9a83-4ba7-9825-1e1a50a5f8e1.png)

**Additional context**
Latest version of aliucord and themer

**Comment by MrSpidercat** [comment]
Sorry that I didn't see this sooner.
This is not an issue with the theme, this is an issue with the Themer plugin or your ROM. You ***can*** fix this issue by installing Aliucord with a Debuggable manifest, but this disables a lot of Android optimizations and your app may run slower.

**Comment by SilverMarcs** [comment]
I see. Can I somehow instead make this theme work with Discord themer LSposed module?

**Comment by MrSpidercat** [comment]
Unfortunately I don't know if it works on teh LSposed module, I haven't yet tried. If it doesn't currently work, there's probably no way to get it working.

I'm currently using a Pixel 6 Pro and the only way any theme works on my phone is to just use Aliucord in debuggable, I don't really experience any issues using it this way though.

**Comment by SilverMarcs** [comment]
Uh can you further tell what you mean by debugging mode? Install Aliucord with the developer mode box checked in aliucord installer?

**Comment by MrSpidercat** [comment]
It means installing Aliucord using a manifest file that has android:debuggable set to true.

You can make Aliucord debuggable by
1. [Downloading the debuggable Manifest from the Aliucord   repo](https://github.com/Aliucord/Aliucord/raw/main/.assets/AndroidManifest-debuggable.xml), rename the file to AndroidManifest.xml and transfer it into the Aliucord folder on your phone.
2. [Download Injector.dex from the Aliucord repo](https://github.com/Aliucord/Aliucord/raw/builds/Injector.dex), move the Injector.dex into the Aliucord folder
3. Create a blank file in Aliucord/.debuggable, then check install with Injector.dex from storage in Aliucord Installer settings


The debuggable manifest that's currently in the Aliucord repo didn't work when I tried, I had to make my own. I have a script for doing this on my GitHub profile, it doesn't work on Windows though. If you've tried the one provided by the Aliucord devs and it doesn't work, I can provide you with the one I created.

**Comment by SilverMarcs** [comment]
I would appreciate you sharing the one you created, thanks

**Comment by MrSpidercat** [comment]
https://drive.google.com/file/d/1l6lm0zFOHsX-c3iYxFQfB4RhrtrctsfP/view?usp=sharing

**Comment by SilverMarcs** [comment]
This worked successfully. May I know if it would be possible to have the statusbar+appbar use the same secondary bg color instead of only the statusbar like it is now? Imo only having the statusbar a different color looks a little off

**Comment by MrSpidercat** [comment]
What do you mean by appbar?

**Comment by MrSpidercat** [comment]
The server list color and statusbar are supposed to match because theres a little transition curve, you can set the status bar color manually if you'd like with the statusbar simple string.

**Comment by MrSpidercat** [comment]
You can see here how the status bar color is supposed to blend in with the rest of the UI
![Screenshot_20220224-082930](https://user-images.githubusercontent.com/41306077/155533367-2c4b84cc-f1d3-45a3-b0ff-b189598e40fa.png)

**Comment by SilverMarcs** [comment]
By appbar, I mean the area below the statusbar that houses the channel name. ie the area above the blue line. I am proposing matching the color used by statusbar and the appbar. 

![Screenshot_20220224-193632_Aliucord](https://user-images.githubusercontent.com/77480421/155534787-704872b5-a1a4-4ba3-9d01-fd5bfd4573c0.png)

The reason I suggest this is to follow design guidelines. For example, in gmail, the appbar and statusbar above the blue line have the same color.

![Screenshot_20220224-193802_Gmail](https://user-images.githubusercontent.com/77480421/155535015-7889e63a-4146-46d1-8586-33b2ed7225a4.png)


So essentially, the statusbar color should stay as it is right now. And the appbar color should match the color of the statusbar.

**Comment by MrSpidercat** [comment]
I still feel it'd throw off Discords design language by doing this. Since the default Discord theme also doesn't follow this.

**Comment by SilverMarcs** [comment]
Yes. But discord still has a different color on appbar compared to the chatpage to **differentiate** the two elements. 
Perhaps a different color for the appbar entirely, similar to what discord does?
Only a suggestion, rest is upto you.
